docs/websockets.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+---
+
+Realtime Messages With WebSockets
+---------------------------------
+
+It's easy to set up WebSockets with Suave.
+
+First, define a function that takes `WebSocket` and `HttpContext` typed parameters, and returns a socket computation expression:
+
+{% highlight fsharp %}
+let ws (webSocket : WebSocket) (context: HttpContext) =
```

Comment:
What do you mean by the `open`-statements? An introduction paragraph? (sorry, non-native english speaker here :) )